keyboards/split75/README.md

```diff
@@ -0,0 +1,26 @@
+# split75
+
+![split75](https://i1.wp.com/mechboards.co.uk/wp-content/uploads/2019/10/HTB1qf0aSOLaK1RjSZFxq6ymPFXae.jpg)
+
+This is a port of the QMK firmware for the split75 keyboard, also known as the ve.a taobao split clone. It runs on
+ps2avrGB firmware, like the [ps2avrGB keyboard](https://www.keyclack.com/product/gb-ps2avrgb/) or the ones sold
+by [Winkeyless](http://winkeyless.kr/product/ps2avrgb-parts/).
+
+Original work performed by Michael L. Walker, all credit goes to him. I've tidied up the keymap (removed 3 extra keys), a couple of functions, info.json and this Readme.
+
+* Keyboard Maintainer: [Michael L. Walker](https://github.com/walkerstop)
+* Hardware Availability: https://a.aliexpress.com/_BT9raf https://item.taobao.com/item.htm?id=556970472669&toSite=main
+
+Make example for this keyboard (after setting up your build environment):
+
+    make split75:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
+
+    make split75:default:flash
+	or
+	bootloadHID -r PATH_TO_FOLDER\split75_default.hex
```

Comment:
```suggestion
```

Reply:
I've argued before in this PR: is there a harm in keeping this? I'd argue it's nice to have a backup option in case the first one doesn't work for whatever reason. + this is actually my preferred way of flashing since QMK one takes longer because of recompiling every time I flash

Reply:
Fair enough, I've just tweaked it a bit to fix the indentation. `PATH_TO_FOLDER` isn't really needed here either as both commands should assume you're in the qmk_firmware root directory.